src/press/migrations/0014_auto_20171127_1502.py

```diff
@@ -16,7 +16,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='press',
             name='homepage_preprints',
-            field=models.ManyToManyField(to='submission.Article'),
+            field=models.ManyToManyField(to='submission.Article', blank=True),
```

Comment:
I might be missing something but these changes don't seem to match the description for this commit: https://github.com/BirkbeckCTP/janeway/pull/3124/commits/07de9335c1e351529068181d33e448ade8928932

Reply:
We discussed this on a call, and it is a safe change from another issue.